mne/bem.py

```diff
@@ -1108,12 +1107,112 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
 
     # Show computed BEM surfaces
     if show:
-        plot_bem(subject=subject, subjects_dir=subjects_dir,
+        plot_bem(subject=subject, sub1jects_dir=subjects_dir,
                  orientation='coronal', slices=None, show=True)
 
     logger.info('Created %s\n\nComplete.' % (fname_head,))
 
 
+def convert_surface(fname_in, fname_out, T1_mgz, useLAS=False):
+    """Convert surface.
+
+    Parameters
+    ----------
+    fname_in : str
+        Path to surface in FreeSurfer surf format.
+    fname_out : str
+        File to write.
+    T1_mgz : str
+        Full path to T1.mgz file.
+    useLAS :
+        Flip x axis.
```

Comment:
Do we need to expose this? Even if it's in the C tools lets not expose it unless we need to. You can just set `useLAS = False` below and add a comment saying we can add the option later if people want to use it. Besides, I think we'd probably want `coords='RAS' | 'LAS'` or something if we were to implement it